package.json

```diff
@@ -26,7 +26,7 @@
     "@angular/router": "~9.1.2",
     "@knora/api": "file:.yalc/@knora/api",
     "jdnconvertiblecalendar": "^0.0.5",
-    "jdnconvertiblecalendardateadapter": "^0.0.12",
+    "jdnconvertiblecalendardateadapter": "^0.0.13",
```

Comment:
@flavens I hope it works :-)

Reply:
Did it work without version conflicts this time?

Reply:
yes

Reply:
cool, so it seems we know what we are doing ;-)